### PR TITLE
Add 1.5.17 post

### PR DESCRIPTION
**Pull request by Adam-**

### CODE REVIEW

File: src/_posts/2019-03-21-12-00-1.5.17-Release.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+title: '1.5.17 Release'
+description: 'NPC unaggro timer and chat filter'
+author: Adam
+---
+
+A NPC unaggressive timer plugin was added, which shows how long until NPCs near
```
Comment (Alexsuperfly) [inline src/_posts/2019-03-21-12-00-1.5.17-Release.md]:
```suggestion
A NPC Aggression Timer plugin was added, which shows how long until NPCs near
```

Consistency with what the name of the plugin will be in the client

File: src/_posts/2019-03-21-12-00-1.5.17-Release.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+you will become unagressive, and how far away you have to walk before they will
```
Comment (Alexsuperfly) [inline src/_posts/2019-03-21-12-00-1.5.17-Release.md]:
```suggestion
you will become unaggressive, and how far away you have to walk before they will
```

spelling

File: src/_posts/2019-03-21-12-00-1.5.17-Release.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+become agressive once again.
```
Comment (Alexsuperfly) [inline src/_posts/2019-03-21-12-00-1.5.17-Release.md]:
```suggestion
become aggressive once again.
```

spelling

File: config-overrides.js [hunk 4]
```diff
@@ -9,6 +9,7 @@ const fs = require('fs')
 const path = require('path')
 const HtmlWebpackPlugin = require('html-webpack-plugin')
 const PrerenderSPAPlugin = require('prerender-spa-plugin')
+const Renderer = PrerenderSPAPlugin.PuppeteerRenderer
```
Comment (Nightfirecat) [inline config-overrides.js]: Are the changes to this file intentional? Seems like they shouldn't be part of a release post PR... Rebase is likely needed.

File: src/_posts/2019-03-21-12-00-1.5.17-Release.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
+---
+title: '1.5.17 Release'
+description: 'NPC unaggro timer and chat filter'
```
Comment (Nightfirecat) [inline src/_posts/2019-03-21-12-00-1.5.17-Release.md]: Same as post body, this should use consistent plugin name.

File: src/_posts/2019-03-21-12-00-1.5.17-Release.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+^Player
+[0-9.]\s*usd
+[0-9][0-9]:[0-9][0-9]:[0-9][0-9]
+doubling
+tripling
+I don't scam
+Fire cape & accounts
+```
+
+is highly effecive against the current spam.
```
Comment (Nightfirecat) [inline src/_posts/2019-03-21-12-00-1.5.17-Release.md]:
```suggestion
is highly effective against the current spam.
```

File: src/_posts/2019-03-21-12-00-1.5.17-Release.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+![chatfilter](/img/blog/1.5.17-Release/chatfilter.png)
+
+We may open up a channel on Discord for users to share patterns similar to bank
+tabs.
+
+Even though this is a rather crude solution to the spam epidemic in RS, and
+spamfilters are in general a solved problem, any solution which is much more
```
Comment (Nightfirecat) [inline src/_posts/2019-03-21-12-00-1.5.17-Release.md]:
```suggestion
spam filters are in general a solved problem, any solution which is much more
```

**Pull request merged**